EventsExpress.Test/ServiceTests/EventServiceTest.cs

```diff
@@ -40,15 +39,15 @@ internal class EventServiceTest : TestInitializer
         private EventLocation eventLocationMapSecond;
         private EventLocation eventLocationOnline;
 
-        private Guid userId = Guid.NewGuid();
         private Guid secondUserId = Guid.NewGuid();
         private Guid eventId = Guid.NewGuid();
         private Guid eventLocationIdMap = Guid.NewGuid();
-        private Guid eventLocationIdOnline = Guid.NewGuid();
-        private Guid eventCategoryId = Guid.NewGuid();
         private Guid eventLocationIdMapSecond = Guid.NewGuid();
+        private Guid userId = Guid.Parse("2eee6760-db3a-4f0d-8eb9-7c9ccac51092");
```

Comment:
Any specific reason to use Parse? If not, please, use Guid.NewGuid, instead of Parse.

Reply:
These particular GUIDs are shared with [https://github.com/ita-social-projects/EventsExpress/pull/1070/files#diff-454041271568dfb3d2529f8e63c5aa00d57147e4a1625c80bacd47fedf6f1291R23-R30](test case source) for [https://github.com/ita-social-projects/EventsExpress/pull/1070/files#diff-c818b005acef71457c15768380ba26857922710760293ed0658838e98ac04454R556-R566](GetAll filters unit test). As these fields are private and non-static, currently this seems the only way to share them

Reply:
We discussed this on meeting, test fixture for EventService needs refactoring to extract fake data into separate class, which then can be referenced by the fixture and case sources. This may be done as a separate task